<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stringTab">
      <attribute name="title">
       <string>String</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="paramK">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="maximum">
            <double>10000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Stiffness:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="paramZ">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000000000000000</double>
           </property>
           <property name="value">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Damping:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="paramM">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Masses:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="zParamN"/>
         </item>
         <item row="1" column="1">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>String name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="mCB">
           <property name="text">
            <string>indexed param</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="kParamN"/>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="mParamN"/>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="kCB">
           <property name="text">
            <string>indexed param</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="zCB">
           <property name="text">
            <string>indexed param</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QPushButton" name="genString">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Generate !</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="intDmp">
           <property name="text">
            <string>Internal Damping?</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="paramZosc">
           <property name="decimals">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QCheckBox" name="zoscCB">
           <property name="text">
            <string>indexed param</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLineEdit" name="zoscParamN"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Length:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="strLength">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>800</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="membraneTab">
      <attribute name="title">
       <string>Membrane</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
